zerver/lib/import_realm.py

```diff
@@ -689,6 +689,46 @@ def bulk_import_model(data: TableData, model: Any, dump_file_id: Optional[str] =
         logging.info("Successfully imported %s from %s[%s].", model, table, dump_file_id)
 
 
+def fix_user_groups_data(data: TableData) -> None:
+    # Old exports might not contain the "zerver_namedusergroup" data.
```

Comment:
We don't support importing from other versions, but we do need to think about exports from third-party services.

Reply:
Remvoed this.